### ISSUE TITLE
Error: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined

### ISSUE THREAD

**Issue by tekumara** [eclipse-theia/theia#8628]
### Bug Description:

When a python test fails, the Python Test Log contains:
```
Error: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
```

This looks like a theia issue.

### Steps to Reproduce:

1. Visit https://gitpod.io/#https://github.com/tekumara/typed-cookie 
or
```
git clone https://github.com/tekumara/typed-cookie && docker run -it --init -p 3000:3000 -v "$(pwd):/home/project:cached" theiaide/theia-python:latest
```

2. CMD+SHIFT+P `Python Discover Tests`. Enable and install pytest then run the tests. 
3. The popup `There was an error in running the tests.` appears.
4. Open the Python Test Log and scroll to the bottom.

### Additional Information

- Operating System: macOS / gitpod
- Theia Version: 1.5.0 theiaide/theia-python:latest
DIGEST:sha256:f639984d6fa5d1e766f2e112965ac113d3e8aee664c777099e5db6baf3eb6771

**Comment by vince-fugnitto** [comment]
@tekumara can you confirm that the bug is present when using `master` from this repository?
In addition, to confirm it is not the the project's error, confirm the same use-case works properly in vscode.

**Comment by tekumara** [comment]
Thanks @vince-fugnitto looks like this is fixed on master (commit 6bc8d7593).
